src/hooks/useMellowLPVaults/config.ts

```diff
@@ -0,0 +1,61 @@
+import { providers } from 'ethers';
+import { isUndefined } from 'lodash';
+
+const networkConfigurations = {
+  mainnet: {
+    MELLOW_ETH_WRAPPER: '0x07D6D75CA125a252AEf4d5647198446e5EDc5BBa',
+    MELLOW_VAULTS: [
+      {
+        voltzVault: '0xBBC446081cb7515a524D31e4afDB19dfc6BAa124',
+        erc20RootVault: '0xC99c70492Bc15c056813d1ddA95C89Bb285Cdc86',
+        erc20RootVaultGovernance: '0x973495e81180Cd6Ead654328A0bEbE01c8ad53EA',
+      },
+    ],
+  },
+  goerli: {
+    MELLOW_ETH_WRAPPER: '0xcF2f79d8DF97E09BF5c4DBF3F953aeEF4f4a204d',
+    MELLOW_VAULTS: [
+      {
+        voltzVault: '0x1b876d1b5A8636EFe9835D8ed0231c1429cBfc40',
+        erc20RootVault: '0x62E224d9ae2f4702CC88695e6Ea4aA16D0925BdB',
+        erc20RootVaultGovernance: '0x4DCc9Ad7ff5964d13ee4A6932922f1a24f3f8e25',
+      },
+    ],
+  },
+  default: {
+    MELLOW_ETH_WRAPPER: '0x0000000000000000000000000000000000000000',
+    MELLOW_VAULTS: [],
+  },
+};
+
+const getConfig = (): {
+  MELLOW_ETH_WRAPPER: string;
+  MELLOW_VAULTS: {
+    voltzVault: string;
+    erc20RootVault: string;
+    erc20RootVaultGovernance: string;
+  }[];
+  PROVIDER: providers.BaseProvider;
+} => {
+  const network = process.env.REACT_APP_NETWORK;
+
+  if (isUndefined(network)) {
```

Comment:
isUndefined from lodash gives false when input is null.

perhaps use `!network` for this place. I am not sure how the process.env.VARIABLE can be passed. Maybe it can be null somehow :D

Reply:
sure. actually, I have one question here, when network is not included in the config, do we want to throw error or return some kind of default value?

Reply:
ideally we want to have sentry error reported to us... I think that these variables must be passed since our App cannot work otherwise. So I would say add a TODO: next to your throw new Error and we handle it better once our project setup allows.

Reply:
for me this is where context + context variables really shine. We can have <ENVsProvider /> component that can resolve this for us. And we can have a hook useEnvVars() that we use to read from Context in any component basically.

Reply:
Yes, we let's create a task around this in the BAU section where we also mention off-loading other env. variables.

Reply:
sure. tests are failing at this moment because network is not specific. where do we set env. variables for the tests?

Reply:
hmmmm we only have tests for Profile... Why don't you call getConfig() where you needed instead of calling it when file is bundled?

Reply:
ok. I can do it.